backend/src/main/java/com/eyetrade/backend/controller/EventController.java

```diff
@@ -0,0 +1,60 @@
+package com.eyetrade.backend.controller;
+
+import com.eyetrade.backend.model.entity.EventRssFeed;
+import com.eyetrade.backend.model.resource.EventResource;
+import com.eyetrade.backend.security.JwtUserChecker;
+import com.eyetrade.backend.service.EventRssReaderService;
+import com.eyetrade.backend.service.EventService;
+import com.sun.syndication.io.FeedException;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+/**
+ * Created by Emir Gökdemir
+ * on 8 Kas 2019
+ */
+@RestController
+@RequestMapping("/event")
+@Api(value = "Event", tags = {"Operations Related With Event"})
+public class EventController {
+
+    @Autowired
+    private EventRssReaderService rssReaderService;
+
+    @Autowired
+    private EventService eventService;
+
+    @Autowired
+    private JwtUserChecker jwtUserChecker;
+
+    @ApiOperation(value = "Update events bu using a rss source", response = EventRssFeed.class)
```

Comment:
There is a type in "bu" :)

Reply:
What is "bu" :)

Reply:
It is part where "event bu using" exist :D